ts/components/conversation/media-gallery/MediaGallery.md

```diff
@@ -0,0 +1,62 @@
+```jsx
+const DAY_MS = 24 * 60 * 60 * 1000;
+const MONTH_MS = 30 * DAY_MS;
+const YEAR_MS = 12 * MONTH_MS;
+const tokens = ['foo', 'bar', 'baz', 'qux', 'quux'];
+const fileExtensions = ['docx', 'pdf', 'txt', 'mp3', 'wmv', 'tiff'];
+const createRandomMessage = ({startTime, timeWindow} = {}) => (props) => {
+  const now = Date.now();
+  const fileName =
+    `${_.sample(tokens)}${_.sample(tokens)}.${_.sample(fileExtensions)}`;
+  return {
+    id: _.random(now).toString(),
+    received_at: _.random(startTime, startTime + timeWindow),
+    attachments: [{
+      data: null,
+      fileName,
+      size: _.random(1000, 1000 * 1000 * 50),
+    }],
+
+    // TODO: Revisit
```

Comment:
And do what? Maybe also make an issue to track the change?

Reply:
Good catch, this was cruft.